disk/datadog_checks/disk/disk.py

```diff
@@ -65,6 +68,38 @@ def __init__(self, name, init_config, instances):
         self._compile_tag_re()
         self._blkid_label_re = re.compile('LABEL=\"(.*?)\"', re.I)
 
+        self.log.info("entering DISK check")
+
+        if platform.system() == 'Windows':
+            self._manual_mounts = instance.get('create_mounts', [])
+            if not self._manual_mounts:
+                self.log.info("No manual mounts")
+            else:
+                self.log.info("Manual mounts: %d", len(self._manual_mounts))
+                for manual_mount in self._manual_mounts:
+                    self.log.info("mm: %s", manual_mount)
```

Comment:
I think these (and other logs added by this PR) should be either removed, set to debug level or they should be rewritten to be more descriptive: it would be hard to know what `mm` stands for without context.

Reply:
yes.  I was printf debugging.